### PR TITLE
[content update] 742-environment-variables

### PR DESCRIPTION
**Pull request by DiSanchz**
This pull request contains the proposed updates for the building block [Configure Environment Variables](https://tilburgsciencehub.com/building-blocks/store-and-document-your-data/store-data/environment-variables/), as part of the task assigned to Issue #742.

### Main comments on the non-updated version

- The title reflects the content of the building block, but it could be reformulated to further emphasize its usefulness
- While it is reasonably clear what the user can learn in the building block, it would benefit from a brief dedicated paragraph devoted to summarizing learning goals.

### Implemented modifications

- Square brackets removed from "mac/Linux" step 3 for further clarity (these could be a bit misleading). 
- Short learning goals paragraph added at the beginning of the text. Headers were modified accordingly.
- Keywords expanded.
- Some minor Grammarly suggestions implemented
- A new title is proposed to better align the BB with the section where it is contained and emphasize its usefulness.

### CODE REVIEW

**Review by thierrylahaije — APPROVED** [review]

File: content/building-blocks/store-and-document-your-data/store-data/environment-variables.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
 ---
-title: "Configure Environment Variables"
+title: "Store your credentials safely by configuring Environment Variables"
```
Comment (srosh2000) [inline content/building-blocks/store-and-document-your-data/store-data/environment-variables.md:2]: Good one! Just on capitalization: "Safely Store Your Credentials by Configuring Environment Variables"

File: content/building-blocks/store-and-document-your-data/store-data/environment-variables.md [hunk 2]
```diff
@@ -32,7 +34,7 @@ Never upload a script that contains privacy-sensitive information to Github. Eve
 2. In the left sidebar click on "Advanced system settings".
 3. Click on "Environment Variables" in the bottom right.
 4. Create a new "User Variable" (top list) and fill out the "Variable name" and "Variable value".
-5. Double click "OK" twice.
+5. Double-click "OK" twice.
```
Comment (srosh2000) [inline content/building-blocks/store-and-document-your-data/store-data/environment-variables.md:39]: Add a very brief "summary" section with key bullet points in the end maybe?

File: content/building-blocks/store-and-document-your-data/store-data/environment-variables.md (hunk 1 above)
Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/environment-variables.md]: Title updated! 🔝 🔝

File: content/building-blocks/store-and-document-your-data/store-data/environment-variables.md (hunk 2 above)
Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/environment-variables.md:39]: Good point, I just added it at the end right before "see also" 🎯

Comment (DiSanchz) [reply content/building-blocks/store-and-document-your-data/store-data/environment-variables.md:39]: PD: I've seen that in other building blocks the final "Summary" section is below the "See also" one so I moved it to that position.

**Comment by lachlandeer** [comment]
thanks!

**Pull request merged**